telethon/telegram_client.py

```diff
@@ -315,8 +335,8 @@ def sign_in(self, phone=None, code=None,
                 result = self(SignInRequest(phone, phone_code_hash, code))
 
             except (PhoneCodeEmptyError, PhoneCodeExpiredError,
-                    PhoneCodeHashEmptyError, PhoneCodeInvalidError):
-                return None
+                    PhoneCodeHashEmptyError, PhoneCodeInvalidError) as e:
+                raise e  # Hand responsibility to the user
```

Comment:
`raise` is enough to re-raise. But why `except` it in the first place if we do no further processing?